magenta/tools/build.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+#
+# Builds the pip package, and then installs it.
```

Comment:
I think we typically have the copyright header first and follow it with any descriptive comments